pennylane_lightning/core/src/measurements/MeasurementsBase.hpp

```diff
@@ -449,7 +439,33 @@ template <class StateVectorT, class Derived> class MeasurementsBase {
         return outcome_map;
     }
 
+    void
+    setDeviceSeed(const std::optional<std::size_t> &deviceSeed = std::nullopt) {
```

Comment:
could you add docstrings to the new methods?

Reply:
Done